components/SearchSandbox/index.js

```diff
@@ -283,7 +290,8 @@ class SearchSandbox extends Component {
 		const {
 			appId, appName, url, credentials, attribution, customProps,
 		} = this.props; // prettier-ignore
-		const { componentProps, mappings, version } = this.state;
+		// eslint-disable-next-line object-curly-newline
```

Comment:
Why we have disabled these rules? Was it because prettier was messing up formatting?

Reply:
Yes, it was giving eslint warning, solving that warning results in bad formatting

Reply:
Do `//prettier-ignore` instead